### DISCUSSION TITLE
文件导入进去无法自动抓取条目

### DISCUSSION THREAD

**Discussion by 233ji** [zotero-chinese/wiki#306] [求助 / Q&A]
出现“1710273527”错误

**Comment by 233ji** [comment]
![批注 2024-11-22 224934](https://github.com/user-attachments/assets/0823948c-0b58-4446-b769-9c99919d33d6)
![批注 2024-11-23 011400](https://github.com/user-attachments/assets/461736cc-5f24-4956-9d7f-ca22dfd4d71b)
![034e7ac2-0bcf-4de8-95c6-d4fb5a09dfcc](https://github.com/user-attachments/assets/4b522e01-0fde-4e92-81b8-86d10aeb7ea5)
笔记模板也无法下载

**Reply by 233ji** [reply]
”1710273527“是我的错误报告ID

**Reply by northword** [reply]
我们无法访问提交给 Zotero 官方的错误报告，你需要前往 Zotero 官方论坛反馈。
